### ISSUE TITLE
"Short" list syntax is unrecognized

### ISSUE THREAD

**Issue by dlh01** [Automattic/VIP-Coding-Standards#459]
## Bug Description

Variables assigned using short `list()` syntax are reported as undefined.

## Minimal Code Snippet

```php
[ $foo ] = func();
```

## Error Code

`Variable $foo is undefined. (WordPressVIPMinimum.Variables.VariableAnalysis.UndefinedVariable)`

## Environment

| Question                | Answer
| ------------------------| -------
| PHP version             | 7.3.12
| PHP_CodeSniffer version | 3.5.1
| VIPCS version           | 2.0.0

## Additional Context

This syntax was added in PHP 7.1: https://www.php.net/manual/en/migration71.new-features.php#migration71.new-features.symmetric-array-destructuring

## Tested Against `master` branch?

- [x] I have verified the issue still exists in the `master` branch of VIPCS.

**Comment by GaryJones** [comment]
Thanks for reporting this @dlh01.

I'm going to close this one as it should be handled in VIPCS by our [switching to an external version of VariableAnalysis](https://github.com/Automattic/VIP-Coding-Standards/issues/449), and I know that work is under way (by @sirbrillig)  there to make use of PHPCSUtils (by @jrfnl) which includes utility methods which better identifies a short list from a short array.

**Comment by GaryJones** [comment]
Now that #449 is completed to use VariableAnalysis proper, I've confirmed that the provided example above doesn't give an error with the `develop` version of `WordPress-VIP-Go`, so this is now fixed for VIPCS 2.2.0.

**Comment by GaryJones** [comment]
Now that #449 is completed to use VariableAnalysis proper, I've confirmed that the provided example above doesn't give an error with the `develop` version of `WordPress-VIP-Go`, so this is now fixed for VIPCS 2.2.0.
